CMakeLists.txt

```diff
@@ -1149,8 +1130,12 @@ function(TRANSFORM_MAKEFILE_INC INPUT_FILE OUTPUT_FILE)
 
 endfunction()
 
-add_subdirectory(docs)
+if(ENABLE_MANUAL)
```

Comment:
Shouldn't this be USE_MANUAL? Otherwise the target is still created even if perl is not available

Reply:
@Lekensteyn agreed. But not ENABLE_MANUAL will be set to on only if PERL_FOUND. Check here conditions https://github.com/snikulov/curl/blob/e6c15e461bc9a94c0f26d9f3ebde330ce4765539/CMakeLists.txt#L210

Reply:
Ah, nice application of that option, learned something new :) It is supported since at least 2.8.12, so that is one concern less.

Reply:
@Lekensteyn you definitely wrong. This feature supported since 2.6
Ref: https://github.com/Kitware/CMake/blob/v2.6.4/Modules/CMakeDependentOption.cmake

Current PR just make life easy for Windows developers, by making Perl not and not using NROFF even it was not found  (ref #1500).

Regarding FindNROFF.cmake I can create a new task for that and assign it to you. Do you agree?

Reply:
By "at least" I meant that that is the earliest version I checked (recently that became the minimum version, so I did not bother with older versions). It's good.

Regarding FindNROFF.cmake, I will try